components/device-mgt/org.wso2.carbon.device.mgt.common/src/main/java/org/wso2/carbon/device/mgt/common/license/mgt/License.java

```diff
@@ -36,7 +36,7 @@
     private String text;
 
     public License() {
-        this.validFrom = new Date();
+        this.validTo = new Date();
```

Comment:
It was added temporary and removed in commit [1].
[1] https://github.com/wso2/carbon-device-mgt/commit/c41f6c300200f1afe56d2ef494428db2e6cd36cc